From 8b83922c38219e7da4ed8f8ecd03683ff104fc01 Mon Sep 17 00:00:00 2001
From: Peng Ma <peng.ma@nxp.com>
Date: Fri, 15 Mar 2019 09:58:26 +0800
Subject: [PATCH 092/235] ARM: dts: ls1021a: Enable edma transfer for i2c

EDMA interrupts doesn't increase on ls1021atwr with kernel 4.14

Signed-off-by: Peng Ma <peng.ma@nxp.com>
[Xulin: Original patch taken from NXP LSDK-19.06.]
Signed-off-by: Xulin Sun <xulin.sun@windriver.com>
---
 arch/arm/boot/dts/ls1021a.dtsi | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/arch/arm/boot/dts/ls1021a.dtsi b/arch/arm/boot/dts/ls1021a.dtsi
index bb866596a931..b923ee1b2795 100644
--- a/arch/arm/boot/dts/ls1021a.dtsi
+++ b/arch/arm/boot/dts/ls1021a.dtsi
@@ -379,6 +379,9 @@
 			interrupts = <GIC_SPI 88 IRQ_TYPE_LEVEL_HIGH>;
 			clock-names = "i2c";
 			clocks = <&clockgen 4 1>;
+			dma-names = "tx", "rx";
+			dmas = <&edma0 1 39>,
+			       <&edma0 1 38>;
 			fsl-scl-gpio = <&gpio3 23 0>;
 			status = "disabled";
 		};
-- 
2.17.1

